### ISSUE TITLE
Please use /etc/sudoers.d to give superuser rights to the default user.

### ISSUE THREAD

**Issue by charles-plessy** [andsens/build-debian-cloud#43]
Hi Anders,

Could you change plugins/admin-user-tasks/create-user to drop a configuration file in /etc/sudoers.d instead of modifying /etc/sudoers directly ?  This will make smoother upgrades each time /etc/sudoers is modified by upgrade of the sudo package.  Here is an example from Ubuntu, where the default account is "ubuntu".

```
root@ip-10-148-5-107:/etc# cat sudoers.d/90-cloudimg-ubuntu 
# ubuntu user is default user in cloud-images.
# It needs passwordless sudo functionality.
ubuntu ALL=(ALL) NOPASSWD:ALL
```

Cheers,
## 

Charles Plessy
Tsurumi, Kanagawa, Japan

**Comment by andsens** [comment]
Will do, makes a lot more sense. I have a lot on my hands right now, after the 15th I should be able to do it.
Or, if you were to send a pull request? :-)

**Comment by tmatilai** [comment]
This could be closed as it's fixed by #45.

**Comment by andsens** [comment]
Correct.
